Cargo.toml

```diff
@@ -95,8 +96,8 @@ tracing = { version = "0.1.40", default-features = false, features = ["std"] }
 tracing-core = "0.1.32"
 tracing-subscriber = "0.3.18"
 unicode-normalization = "0.1.24"
-uniffi = { version = "0.28.0" }
-uniffi_bindgen = { version = "0.28.0" }
+uniffi = { version = "=0.28.0" }
+uniffi_bindgen = { version = "=0.28.0" }
```

Comment:
`0.28.0` with no SemVer operators resolves to `^0.28.0` which is satisfied by the latest `0.28.3` which Cargo can automatically upgrade to when dependencies are touched or the lockfile is regenerated, as per the [docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html). This version has a breaking change in the method `generate_bindings`, which breaks xtask. Locking this to `=0.28.0` resolves the issue for now.